frontend/projects/dipa-frontend/src/app/modules/gantt/chart/chart-elements/MilestonesArea.ts

```diff
@@ -170,6 +170,9 @@ export class MilestonesArea implements IChartElement {
             this.onSelectMilestone(d);
 
           }
+          else {
+            this.onSelectMilestone(d);
+          }
```

Comment:
wenn
`this.onSelectMilestone(d);`
immer ausgeführt wird, kann es auch nach dem if stehen und das else kann man sich sparen